recipe/meta.yaml

```diff
@@ -7,20 +8,21 @@ package:
 source:
   fn: traitlets-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/t/traitlets/traitlets-{{ version }}.tar.gz
-  md5: d0725f9818bd9e112a6531da26d433b8
+  sha256: {{ sha256 }}
 
 build:
   number: 0
-  script: pip install --no-deps .
+  script: python setup.py install
```

Comment:
@ocefpaf is there a link to the discussion of removing pip from Python in conda-forge? I see that `conda install python` still gets pip locally when everything's coming from conda-forge, so it seems weird that the dependencies in the recipe resolve differently than they would for a normal conda env.

anaconda-installed Python lacked pip in the beginning, which caused endless headaches and user problems, which eventually got fixed when `conda install python` included pip as users expect. I would like to make sure that conda-forge doesn't make the same mistake again.